src/Component/Home/Products/index.jsx

```diff
@@ -31,8 +33,8 @@ function Products() {
         <div className="product-homepage__main">
           {productsList &&
             productsList.map((item) => (
-              <div className="mb-4 col-12 col-sm-6 col-md-3">
-                <ProductItem key={item.id} data={item} />
+              <div key={item.id} className="mb-4 col-12 col-sm-6 col-md-3">
+                <ProductItem data={item} />
```

Comment:
```suggestion
                <ProductItem {...item} />
```
Truyền như này cũng được nhé
Lúc này props của ProductItem sẽ đơn giản là
`function ProductItem({ id, title, price, image, category })`